config/routes.rb

```diff
@@ -5,6 +5,10 @@
       get '/account/sign_up' => 'account_sign_up#new', as: 'new_sign_up'
       post '/account/sign_up' => 'account_sign_up#create'
       get '/', to: 'splash#index'
+
+      # pending https://github.com/projecthydra-labs/hyrax/issues/376
+      get '/dashboard', to: redirect('/')
```

Comment:
what if you actually want to go to the dashboard?  won't this prevent that from happening?